labelbox/schema/model_run.py

```diff
@@ -180,6 +181,39 @@ def delete_model_run_data_rows(self, data_row_ids):
             data_row_ids_param: data_row_ids
         })
 
+    @experimental
+    def assign_data_rows_to_split(self,
+                                  data_row_ids,
+                                  split,
+                                  timeout_seconds=60):
+        valid_splits = ["TRAINING", "TEST", "VALIDATION"]
```

Comment:
**Thought:** Would it make sense to have this as an enum of Dataset or ModelRun? Unlikely, but if we add more to the splits or use this elsewhere

Reply:
It prob should be an enum. My thought was that 1) we aren't consistent with our usage of enums so this isn't inconsistent with our existing lack of a pattern and 2) if we require an enum it is another thing for users to import. I do want to chat about this with you and Kevin today if you have some time. Just more generally what we want to do.

Reply:
sgtm!